lib/ffmpeg/vaapi/encoder.py

```diff
@@ -31,8 +31,11 @@ def profile(self):
   @property
   def qp(self):
     def inner(qp):
-      if self.codec in [Codec.VP8, Codec.VP9, Codec.AV1]:
+      if self.codec in [Codec.VP8, Codec.VP9]:
         return f" -global_quality {qp}"
+      if self.codec in [Codec.AV1]:
+        if "ICQ" == self.rcmode:
```

Comment:
@uartie  @Bin-CI  actually, I also though it to be lowercase before this PR submitted. but found in the file of lib/ffmpeg/encoderbase.py
class Encoder(PropertyHandler, BaseFormatMapper):
      rcmode        = property(lambda s: s.props['rcmode'].upper())

the rcmode will be auto transferred to uppercase for both ffmpeg-vaapi/qsv

Reply:
@FocusLuo you are right!  The `Encoder` class responsibility is to convert the options to command-line syntax when accessed via `property` members.  So, yes, uppercase is correct here since you're accessing through the property field (`self.rcmode`).  The original can be accessed through `self.props["rcmode"]`, which is lowercase and comes from the `EncoderTest` variables.  Thus, in `EncoderTest`, the `self.rcmode` would be lowercase.  In general, it would be better to use the `self.props["rcmode"]` access in `Encoder`, internally, because some properties will have the command-line option included.

Reply:
I'm ok with your current solution for now, since we also access `self.codec` property, internally, without consideration that it could change in the future to include additional command-line options.

I will think about a refactor later to make it better to avoid this kind of confusion.